.github/workflows/tests.yml

```diff
@@ -80,12 +80,18 @@ jobs:
         run: rustup component add clippy
       - name: Run clippy
         run: cargo clippy --release -- -D warnings
-      - name: Tests
-        run: time cargo test --all --release --verbose --locked
-      - name: Benchmarking tests
-        run: cd node && cargo test --release --features runtime-benchmarks -p pallet-exchange-benchmarking -p pallet-xyk -p pallet-multi-payment-benchmarking -p pallet-claims
```

Comment:
benchamrking tests are removed intentionally?

Reply:
yes, removed. Benchmarks tests are part of the other step and included in the coverage too.